### PR TITLE
Danielle/reconcile

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: pkg/actuators/machine/vms.go [hunk 1]
```diff
@@ -1,27 +1,309 @@
...
+	requeueAfterFatalSeconds = 180
+	masterLabel              = "node-role.kubernetes.io/master"
+)
+
+// Reconciler runs the logic to reconciles a machine resource towards its desired state
+type Reconciler struct {
+	*machineScope
+}
+
+func newReconciler(scope *machineScope) *Reconciler {
```
Comment (nirarg) [inline pkg/actuators/machine/vms.go:27]: Change to newVmManager

File: pkg/actuators/machine/vms.go [hunk 2]
```diff
@@ -1,27 +1,309 @@
...
-func createVM(virtualMachine *kubevirtapiv1.VirtualMachine, underkubeclient kubevirtclient.Client, namespace string) (*kubevirtapiv1.VirtualMachine, error) {
-	return underkubeclient.CreateVirtualMachine(namespace, virtualMachine)
+const (
+	requeueAfterSeconds      = 20
+	requeueAfterFatalSeconds = 180
+	masterLabel              = "node-role.kubernetes.io/master"
+)
+
+// Reconciler runs the logic to reconciles a machine resource towards its desired state
+type Reconciler struct {
```
Comment (nirarg) [inline pkg/actuators/machine/vms.go:23]: This is not Reconciler, change to vm_manager :-)
can be lower case if accessed only from within the package

Comment (bardielle) [reply pkg/actuators/machine/vms.go]: We decided to change it to providerVM

File: pkg/actuators/machine/vms.go (hunk 1 above)
Comment (bardielle) [reply pkg/actuators/machine/vms.go]: You right I missed it

**Pull request closed without merging**